examples/resources/datadog_monitor/resource.tf

```diff
@@ -18,7 +18,7 @@ resource "datadog_monitor" "foo" {
   renotify_interval = 60
 
   notify_audit = false
-  timeout_h    = 60
+  timeout_h    = 18
```

Comment:
Do we actually want to have this in the example? There's a bunch of fields that aren't populated here already and I don't know if the autoresolve timeout is a behaviour that people want to have by default.

Reply:
fair enough, i've removed it entirely